Sources/PackageDescription/BuildSettings.swift

```diff
@@ -127,10 +130,8 @@ public struct CSetting: Encodable {
     ///
     /// The path must be a directory inside the package.
     ///
-    /// - Since: First available in PackageDescription 5.0
```

Comment:
Should this and the other "Since"s really get removed?  In the corresponding PR 1, it seemed that they were getting preserved since they convey useful information.

Reply:
Added all of them back in.